Changelog.md

```diff
@@ -12,6 +12,7 @@
 
 - 60FPS: Refactor 60FPS battle menu fix and fix ESUI compatibility ( https://github.com/julianxhokaxhiu/FFNx/pull/597 )
 - Ambient: Allow ambient effects to playback in fields that use movies as background
+- Common: Fix a softlock when trying to change in-game Controller setting to "Normal" (by disabling this option)
```

Comment:
Anything that is related to Steam used to have its own section on the Changelog, can you please move this line into the dedicated section? See https://github.com/julianxhokaxhiu/FFNx/blob/master/Changelog.md#ff7-steam as an example :) thank you!

Reply:
@julianxhokaxhiu Done.